packages/cli/src/commands/run.js

```diff
@@ -0,0 +1,93 @@
+import fs from 'fs'
+import path from 'path'
+
+import babelRequireHook from '@babel/register'
+import Listr from 'listr'
+import VerboseRenderer from 'listr-verbose-renderer'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from 'src/lib'
+import c from 'src/lib/colors'
+
+babelRequireHook({
+  extends: path.join(getPaths().api.base, '.babelrc.js'),
+  extensions: ['.js', '.ts', '.tsx', '.jsx'],
+  only: [getPaths().api.base],
+  plugins: [
+    [
+      'babel-plugin-module-resolver',
+      {
+        alias: {
+          src: getPaths().api.src,
+        },
+      },
+    ],
+  ],
+  ignore: ['node_modules'],
+  cache: false,
+})
+
+const { db } = require(path.join(getPaths().api.lib, 'db'))
+
+const runScript = async (scriptPath) => {
+  const script = await import(scriptPath)
+  const startedAt = new Date()
+  await script.default({ db })
+  const finishedAt = new Date()
+
+  return { startedAt, finishedAt }
+}
+
+export const command = 'run'
+export const description = 'Run your script'
+export const builder = (yargs) => {
+  yargs
+  .positional('name', {
```

Comment:
What do you think of allowing an array of names? Would be nice to be able to run multiple scripts in one go!

Reply:
Oh really? Ive personally never had that use case. Would that be names comma seperated?

Reply:
It's also handled with unix commands much easier by doing `yarn rw run <cmd1> && yarn rw run <cmd1>` - so they get executed one after the other

Reply:
Hmm, it actually makes more sense to have them run individually if we're passing in params.